series-tmdb.en.yaml

```diff
@@ -0,0 +1,2683 @@
+    # https://www.themoviedb.org/tv/60833
+    # https://www.thetvdb.com/dereferrer/series/278329
+    # https://www.imdb.com/title/tt3874528/
+    synonyms:
+      - "Mahouka"
+      - "The Irregular of a Magic High School"
+      - "A Magic Senior High School's Low-Ranked Student"
+      - "A Poorly Performing Student of a Magic High School"
+      - "The Poor Performing Student of a Magic High School"
+      - "Mahouka Koukou no Rettousei"
+      - "Mahōka Kōkō no Rettōsei"
+      - "魔法科高校の劣等生"
+    seasons:
+      - season: 1
+        anilist-id: 20458
+      - season: 2
+        anilist-id: 112300
+      - season: 3
+        anilist-id: 143271
+
+  - title: "The King's Avatar"
+    # https://www.themoviedb.org/tv/71194
+    # https://www.thetvdb.com/dereferrer/series/326270
+    # https://www.imdb.com/title/tt6859260/
+    synonyms:
+      - "Full-Time Expert"
+      - "Quan Zhi Gao Shou"
+      - "Quanzhi Gaoshou"
+      - "全职高手 动画版"
+    seasons:
+      - season: 1
+        anilist-id: 98861
+      - season: 2
+        anilist-id: 102997
+      - season: 3
+        anilist-id: 137735
+
+  - title: "The Master of Ragnarok & Blesser of Einherjar"
+    # https://www.themoviedb.org/tv/80670
+    # https://www.thetvdb.com/dereferrer/series/347294
+    # https://www.imdb.com/title/tt9042408/
+    synonyms:
+      - "Hyakuren no Haou to Seiyaku no Ikusa Otome"
+      - "Hyakuren no Haō to Seiyaku no Valkyria"
+    seasons:
+      - season: 1
+        anilist-id: 101289
+
+  - title: "The Misfit of Demon King Academy"
+    # https://www.themoviedb.org/tv/97617
+    # https://www.thetvdb.com/dereferrer/series/375927
+    # https://www.imdb.com/title/tt12432936/
+    synonyms:
+      - "Demon King Academy"
+      - "Maou Gakuin no Futekigousha"
+      - "The Greatest Demon Lord Is Reborn as a Typical Nobody"
+      - "Demon King Mismatched School ~The Founder of the Strongest Demon King in History, He Arrives at School as Descendants After Reincarnating~"
+      - "Maou Gakuin no Futekigousha ~Shijo Saikyou no Maou no Shiso, Tenseishite Shison-tachi no Gakkou e Kayou~"
+      - "Maou Gakuin no Futekigousha: Shijou Saikyou no Maou no Shiso, Tensei Shite Shison-tachi no Gakkou e Kayou"
+      - "Maō Gakuin no Futekigōsha ~Shijo Saikyō no Maō no Shiso, Tenseishite Shison-tachi no Gakkō e Kayō~"
+      - "Maō Gakuin no Futekigōsha: Shijō Saikyō no Maō no Shiso, Tensei shite Shison-tachi no Gakkō e"
+      - "The Misfit of Demon King Academy: History's Strongest Demon King Reincarnates and Goes to School with His Descendants"
+      - "The Unqualified Student at the Demon Lord Academy ~ The Strongest Demon Lord in History, the Founder, Reincarnates and Attends a School With His Descendants ~"
+      - "The Unqualified Student of Demon Lord Academy ~The Strongest Demon Lord in History, the Founder, Attends the School of his Descendants After Reincarnation~"
+      - "魔王学院"
+      - "🈟魔王学院の不適合者"
```

Comment:
sort.py turns `🈟` into `\U0001F21F`. Not sure if that causes problems or not.

Reply:
If anything, it detected an odd symbol that for some reason is part of the Synonyms, which is good, removed.